### PR TITLE
PROD-652 - Create album button issue for group - Fixed

### LINKED ISSUE #3261: Create Albums in Groups will disappear once the Enable Albums in Profiles settings is disabled
**Describe the bug**
Create Albums button in Groups will disappear once the "Enable Albums in Profiles" settings is disabled. See this video: http://somup.com/crlto70WXG

**To Reproduce**
1. Make sure Create Albums is functional on Groups
2. In BuddyBoss > Settings > Media > Photos > Untick Enable album in Profiles
3. View Group, see error that Create Album button is removed

**Expected behavior**
"Enable album in Profiles" option should not interfere the group album 

**Environment**
Local, Dev, Client site

**Support ticket links**
https://secure.helpscout.net/conversation/1736049160/179761/

**Jira issue** : [PROD-652]

[PROD-652]: https://buddyboss.atlassian.net/browse/PROD-652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### PR DESCRIPTION
**Pull request by mahedihasannoman**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes #3261  .

### How to test the changes in this Pull Request:

1. Make sure "Enable albums in groups" option is enabled from BuddyBoss > Settings > Media
2. In BuddyBoss > Settings > Media > Photos > Untick Enable album in Profiles option.
3. See the Create Album button is showing in Group

### Proof Screenshots or Video

https://share.vidyard.com/watch/TFedB71eHnp64dwmsBfyhJ?

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

**Jira issue** : [PROD-652]

[PROD-652]: https://buddyboss.atlassian.net/browse/PROD-652?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php [hunk 1]
```diff
@@ -21,7 +27,9 @@
 	</div>
 
 	<?php
-	if ( bp_is_profile_albums_support_enabled() ) {
+	if ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) {
```
Comment (dipskakadiya) [inline src/bp-templates/bp-nouveau/buddypress/media/albums.php:30]: @mahedihasannoman Here we can use single if statement with **OR**. As a content statement for if and else is same.

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php [hunk 2]
```diff
@@ -10,7 +10,13 @@
 	<div class="bb-media-actions-wrap album-actions-wrap">
 		<h2 class="bb-title"><?php esc_html_e( 'Albums', 'buddyboss' ); ?></h2>
 		<?php
-		if ( bp_is_profile_albums_support_enabled() ) {
+		if ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) {
```
Comment (dipskakadiya) [inline src/bp-templates/bp-nouveau/buddypress/media/albums.php:13]: @mahedihasannoman Here we can use single if statement with OR. As a content statement for if and else is same.

**Review by dipskakadiya — CHANGES_REQUESTED** [review]

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php (hunk 1 above)
Comment (mahedihasannoman) [reply src/bp-templates/bp-nouveau/buddypress/media/albums.php]: Done

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php (hunk 2 above)
Comment (mahedihasannoman) [reply src/bp-templates/bp-nouveau/buddypress/media/albums.php]: Done

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
 	<div class="bb-media-actions-wrap album-actions-wrap">
 		<h2 class="bb-title"><?php esc_html_e( 'Albums', 'buddyboss' ); ?></h2>
 		<?php
-		if ( bp_is_profile_albums_support_enabled() ) {
+		if ( ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) || bp_is_profile_albums_support_enabled() ) {
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/media/albums.php:13]:
```suggestion
		if ( ( bp_is_group() && bp_is_group_albums_support_enabled() ) || bp_is_profile_albums_support_enabled() ) {
```

File: src/bp-templates/bp-nouveau/buddypress/media/albums.php [hunk 4]
```diff
@@ -21,7 +21,7 @@
 	</div>
 
 	<?php
-	if ( bp_is_profile_albums_support_enabled() ) {
+	if ( ( bp_is_group() &&  bp_is_group_albums_support_enabled() ) || bp_is_profile_albums_support_enabled() ) {
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/buddypress/media/albums.php:24]:
```suggestion
	if ( ( bp_is_group() && bp_is_group_albums_support_enabled() ) || bp_is_profile_albums_support_enabled() ) {
```

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need PHPCS

**Review by dipskakadiya — APPROVED** [review]

**Review by KartikSuthar — APPROVED** [review]

**Pull request merged**